src/btree/bt_sync.c

```diff
@@ -227,6 +229,10 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
         __wt_gen_next_drain(session, WT_GEN_EVICT);
         btree->syncing = WT_BTREE_SYNC_RUNNING;
 
+        /* Add in lookaside reconciliation for standard files. */
+        if (!F_ISSET(btree, WT_BTREE_LOOKASIDE) && !WT_IS_METADATA(btree->dhandle))
+            rec_flags |= WT_REC_LOOKASIDE;
```

Comment:
Can we switch to one model of "is this a special file"? I'd suggest `WT_IS_LOOKASIDE`.

Reply:
Added WT_IS_LOOKASIDE macro.